### PR TITLE
Palindrome

### PR DESCRIPTION
**Pull request by Diaboliq**
Given a string of characters, determine if it's a Palindrome or not

### CODE REVIEW

File: Palindrome [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package work;
+
+import java.util.Scanner;
+
+public class Palindrome {
+
+    static boolean isPalindrome(String word) {
```
Comment (CristianManic) [inline Palindrome:7]: in this case we don't care about this, but for future please get used to set an access modifier

File: Palindrome [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        int a = 0;
+        int b = word.length() - 1;
```
Comment (CristianManic) [inline Palindrome:9]: please give more meaningful names to your variables

File: Palindrome (hunk 1 above)
Comment (Diaboliq) [reply Palindrome]: Sure

File: Palindrome (hunk 2 above)
Comment (Diaboliq) [reply Palindrome]: Variable names changed

**Review by CristianManic — APPROVED** [review]

**Pull request merged**